macos/xcode/FluentUI.xcodeproj/project.pbxproj

```diff
@@ -981,13 +981,15 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 8F69C7B22295FABD009F69C0 /* FluentUI_testapp.xcconfig */;
 			buildSettings = {
+				MACOSX_DEPLOYMENT_TARGET = 10.14;
```

Comment:
This should be coming from the xcconfig, and need not be specified here I thought? I guess we need to update the minimum OS version of the test app as well? Looks like line 982 specifies the correct xcconfig

Reply:
I still get all kinds of availability warnings despite setting the deployment target for the test app target, so wherever the setting is currently specified it is not correct.  I guessed that it might need to be set project-wide but I do not actually know.  I will gladly remove all this stuff if we can figure out how to set the minimum OS.

Reply:
Fixed. I must have picked up a change to the minimum OS version recently and was able to remove this stuff.